.circleci/config.yml

```diff
@@ -477,7 +477,7 @@ jobs:
 
   # This job is used for the e2e quality gate.
   # It must be run before any job which uses the run-all.js script.
-  # The job is skipped in develop, master or RC branches.
+  # The job is skipped in main, master or RC branches.
   get-changed-files-with-git-diff:
```

Comment:
The `git-diff-develop.ts` script should be renamed as well, but we can do that in a later PR